tests/functional/api/test_bulk.py

```diff
@@ -58,17 +85,35 @@ def commands(self):
                 },
                 "user_ref",
             ),
-            CommandBuilder.group.upsert(
-                {
-                    "authority": authority,
-                    "authority_provided_id": "name",
-                    "name": "name",
-                },
-                "group_ref",
-            ),
-            CommandBuilder.group_membership.create("user_ref", "group_ref"),
         ]
 
+        for i in range(group_count):
+            commands.append(
+                CommandBuilder.group.upsert(
+                    {
+                        "authority": self.AUTHORITY,
+                        "authority_provided_id": f"id_{i}",
+                        "name": f"name_{i}",
+                    },
+                    f"group_ref_{i}",
+                )
+            )
+
+        for i in range(group_count):
+            commands.append(
+                CommandBuilder.group_membership.create("user_ref", f"group_ref_{i}")
+            )
+
+        return commands
+
+    @pytest.fixture
+    def user(self, db_session):
+        user = User(authority=self.AUTHORITY, _username="username")
+        db_session.add(user)
+        db_session.flush()
```

Comment:
Is this flush necessary?

Reply:
Yes, I seem to recall it was, but I can't remember why. It just didn't seem to show up if not actually pushed through. I'll try it without and see if it's working.

Reply:
Nope, looks like cargo cult nonsense. I probably had an un-related failure at the time and tried this to attempt to fix it.